homeassistant/components/voip/strings.json

```diff
@@ -2,7 +2,10 @@
   "config": {
     "step": {
       "user": {
-        "description": "Receive Voice over IP calls to interact with Assist."
+        "description": "Receive Voice over IP calls to interact with Assist.",
+        "data": {
+          "sip_port": "SIP port"
+        }
```

Comment:
This seems to be misplaced at this point, as this is now part of an options flow, not a config flow.

Reply:
Ah, thanks!